chalice/app.py

```diff
@@ -693,8 +702,11 @@ def _validate_response(self, response):
     def _cors_enabled_for_route(self, route_entry):
         return route_entry.cors is not None
 
-    def _add_cors_headers(self, response, cors):
-        for name, value in cors.get_access_control_headers().items():
+    def _get_cors_headers(self, cors):
+        return dict(cors.get_access_control_headers())
```

Comment:
I don't think we need the `dict()` call because `get_access_control_headers()` return a dict.